### PR TITLE
Niamh25/DOC-5192/X500-member-rules

### PR DESCRIPTION
**Pull request by Niamh25**
Rules added to Vnode section about the X.500 member rules

### CODE REVIEW

File: content/en/platform/corda/5.0/deploying-operating/vnodes/_index.md [hunk 1]
```diff
@@ -12,4 +12,19 @@ section_menu: corda5
...
+
+* The string specified must use the grammar defined in RFC 1779 or RFC 2253.
+* The only supported attributes are CN, OU, O, L, ST, C.
+* Attributes cannot be duplicated and must have a single value.
+* The attributes O, L, C are mandatory.
+* The Organization attribute (O) cannot be blank and must be less than 128 characters.
+* The Locality attribute (L) cannot be blank and must be less than 64 characters.
+* The Country attribute (C) cannot be blank and must be an ISO 3166-1 2-letter country code or "ZZ" to indicate unspecified country.
+* If specified, the State attribute (ST) cannot be blank and must be less than 64 characters.
+* If specified, the Organization unit attribute (OU) cannot be blank and must be less than 64 characters.
```
Comment (yarivus) [inline content/en/platform/corda/5.0/deploying-operating/vnodes/_index.md:29]:
```suggestion
* If specified, the Organization Unit attribute (OU) cannot be blank and must be less than 64 characters.
```

File: content/en/platform/corda/5.0/deploying-operating/vnodes/_index.md [hunk 2]
```diff
@@ -12,4 +12,19 @@ section_menu: corda5
...
-* 
\ No newline at end of file
+* 
+
+## Rules for X.500 Member Names
+
+Each virtual node within an application network is uniquely identified by its X.500 distinguished name. The rules for a valid X.500 name in Corda are the following:
+
+* The string specified must use the grammar defined in RFC 1779 or RFC 2253.
+* The only supported attributes are CN, OU, O, L, ST, C.
```
Comment (yarivus) [inline content/en/platform/corda/5.0/deploying-operating/vnodes/_index.md:22]: I can't see CN description below.

**Review by yarivus — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0/deploying-operating/vnodes/_index.md (hunk 2 above)
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/deploying-operating/vnodes/_index.md:22]:
```suggestion
* The only supported attributes are CN, OU, O, L, ST, and C.
```

File: content/en/platform/corda/5.0/deploying-operating/vnodes/_index.md [hunk 3]
```diff
@@ -12,4 +12,19 @@ section_menu: corda5
...
+* Attributes cannot be duplicated and must have a single value.
+* The attributes O, L, C are mandatory.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/deploying-operating/vnodes/_index.md:24]:
```suggestion
* The attributes O, L, and C are mandatory.
```

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request merged**